### DISCUSSION TITLE
Error importing @truffle/contract v4.6.1

### DISCUSSION THREAD

**Discussion by emilyJLin95** [ConsenSys-archive/truffle#5550] [General Q&A]
<Posted from Discord>

Hi folks! I'm building a dapp and am getting the following error when attempting to import @truffle/contract v4.6.1:

```
/node_modules/@truffle/contract/node_modules/ethers/dist/ethers.min.js 1:484-485
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

I've tried installing ethers directly in package.json in root but am still receiving this error. I've done a decent amount of investigating and have found a few suggestions such as reinstalling node modules, adding ethers in root dependencies, etc. Any advice on a workaround would be much appreciated!

**Comment by apdan510** [comment]
Hey @emilyJLin95 what other modules do you have installed? Can you share your package.json and repo if possible? I remember running into this with some newly updated modules. Thank you!
